### ISSUE TITLE
Party list bugs out intermittently

### ISSUE THREAD

**Issue by CatsEyeXI** [DerpyProjectGroup/topaz#1667]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: base

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

I was able to reproduce this pretty reliably by normally summoning trusts in Dynamis.
Happens pretty frequently (I had never seen this issue before vs. now its pretty consistent)

Basically the party members in the list will disappear/reappear or disappear altogether.

**Comment by RAIST5150** [comment]
This may be related, so not opening a new issue just yet?

Temp items from blue chests in "normal" areas seem to be disappearing from the list intermittently as well.

Noticed today in Shrine of Ru'Avitau.  Healing items, D-wings, etc. would not show up in either of the Items menu lists sometimes.  Could call them manually with /item commands. Right after using them that way or taking an item from a blue chest, the list would sometimes repopulate, but after engaging in a fight they would disappear again for a while.  Sometimes they would just randomly appear again.

![Raist_2022 05 08_114502](https://user-images.githubusercontent.com/73744368/167304276-350b0b46-f49d-4ae2-bbe5-c7d3847e3d21.jpg)
(not there)

![Raist_2022 05 08_114524](https://user-images.githubusercontent.com/73744368/167304306-950a714c-57ac-4e41-bd36-6fb133301ff6.jpg)
(typed /item "Lucid potion I" < me>

![Raist_2022 05 08_114539](https://user-images.githubusercontent.com/73744368/167304331-4c0568bd-177f-488a-b3d1-f5f8019f0344.jpg)
(list repopulated)

**Comment by zach2good** [comment]
Should be fixed in some older commits, ending with: https://github.com/LandSandBoat/server/pull/1738/commits/bb255110533afe388411d88aea42f464fddd08c9
